chainercv/links/model/resnet/resnet.py

```diff
@@ -201,8 +201,9 @@ def __init__(self, n_layer,
             if not _available:
                 warnings.warn('cv2 is not installed on your environment. '
                               'The scores of ResNets reported in the '
-                              'ChainerCV\'s Github page are calculated using '
-                              'OpenCV as the backend. With Pillow as the '
+                              'READMEs in ChainerCV\'s examples are '
```

Comment:
The scores are reported only in https://github.com/chainer/chainercv/tree/master/examples/classification. So `READMEs` -> `README`? Will you add the scores to another README?